### PR TITLE
Setup of testing environment with Rspec, cucumber and coveralls

### PR DESCRIPTION
**Pull request by amandagibson**
#### Description

As a dev team
In order to test our application
We need to setup cucumber and rspec
Changes proposed in this pull request:

* addition of coveralls
* addition of rspec & cucumber
#### What I have learned working on this feature: 
We learned that we need to wait until code is merged from previous PR before starting to work on it again.

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (magnus-thor) [inline .rspec]: change the content in this file to 
```
--require rails_helper
--color
--format documentation
```

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: .rspec (hunk 1 above)
Comment (amanda-mae) [reply .rspec]: done

Comment (magnus-thor) [reply .rspec]: remove the require spec_helper

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: .rspec (hunk 1 above)
Comment (amanda-mae) [reply .rspec]: done

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**